pairtaghighlighter/src/pair_tag_highlighter.c

```diff
@@ -319,6 +321,9 @@ static void findMatchingTag(ScintillaObject *sci, gint openingBracket, gint clos
     gboolean isTagOpening = is_tag_opening(sci, openingBracket);
     gchar *tagName = get_tag_name(sci, openingBracket, closingBracket, isTagOpening);
 
+    if (!tagName)
```

Comment:
The check is not redundant because it checks for `NULL`ness of `tagName`, which neither `is_tag_empty()` nor `findMatchingClosingTag()` nor `findMatchingOpeningTag()` accept.
If you want to can replace with with `tagName ? tagName : ""` or similar as argument to those functions.

And regarding the behavior, does it matter much as we're talking about a tag with no name, which is obviously invalid?  But I don't mind at all a better fix, please improve if you like :)